src/mca/ptl/tcp/ptl_tcp.c

```diff
@@ -1249,17 +1249,24 @@ static pmix_status_t recv_connect_ack(int sd, uint8_t myflag)
         return rc;
     }
     reply = ntohl(u32);
```

Comment:
while here ntohl conversion is present.
It was working because the representation of 0x0 is the same in both LE and BE.

Reply:
Probably also because I'm unaware of anyone actually using mixed endian clusters any more. Still, we try to maintain it just as good practice.

Reply:
But here if one side performs htonl, but another side doesn't it will lead to an issue even on homogeneous cluster